Assets/MixedRealityToolkit.Extensions/Webrtc/Plugins/.gitignore

```diff
@@ -0,0 +1 @@
+*.dll
```

Comment:
do you think that you will want to track anything in the plugin directory? I know that long term there could be some android components. But it may be worth removing all meta files and the plugins directory itself for this check in.

Reply:
A small dev experience thing that i think __is__ nice about having some of this is that the folder structure is already there - makes placing the dlls just a little less painful. 

otherwise, no probably not for the time being.

Reply:
i'd probably opt for not adding the plugins directory here. its probably worth adding to the desription that these files need to go into a plugins directory, but its not super clear where the plugins directory would end up being for most applications. If folks copy your content into their own projects they might already have their own plugins directory that they'd prefer to use.